.travis.yml

```diff
@@ -59,6 +71,8 @@ android:
     - 'android-sdk-license-.+'
 
 script:
+  - git rev-list $(git merge-base HEAD origin/master)..HEAD | xargs -i git clang-format --binary=$(which clang-format-3.8) --style=file {} && git diff --exit-code || { git reset --hard; false; }
+  - git rev-list $(git merge-base HEAD origin/master)..HEAD | xargs -i git diff-tree --no-commit-id --name-only -r {} | grep -E '\.java$' | xargs -r git ls-files | xargs -r java -jar $HOME/gjf.jar -a -i --fix-imports-only && git diff --exit-code || { git reset --hard; false; }
```

Comment:
This seems like it should logically work, but I got some weird results when I didn't have this rebased.